### PR TITLE
Hyundai: Kona torque tune

### LINKED ISSUE #25347: Hyundai: Kona EV 2022 poor lateral tuning
### Describe the bug

Even at Low speeds my Kona EV 2022 takes the bend with very jerky steering. It does take it properly but the jerkiness is probably related to a tuning issue as shared by Shane 

also sometimes when it is taking a round about to get off an express way, it hugs the side too much instead of being centered in the lane (you can also see this in the bend in my video)

you can see in the video about 5:21pm I had to intervene otherwise I would have crashed into the side. 

At about 5:12pm you can see an example of the jerky steering wheel

### Provide a route where the issue occurs

ff973b941a69366f|2022-08-03--17-09-29--0

### openpilot version

0.8.16

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by adeebshihadeh**
closes #25347

### CODE REVIEW

**Comment by sshane** [comment]
@haram-KONA will be very happy if his hybrid was added `CAR.KONA_HEV`

**Comment by haram-KONA** [comment]
If you could also add "kona-hev", it would be greatly appreciated.

Thank you so much ~~~


Windows용 메일<https://go.microsoft.com/fwlink/?LinkId=550986>에서 발송된 메일입니다.

보낸 사람: Shane ***@***.***>
보낸 날짜: 2022년 8월 13일 토요일 오후 3:23
받는 사람: ***@***.***>
참조: ***@***.***>; ***@***.***>
제목: Re: [commaai/openpilot] Hyundai: Kona EV 2022 torque tune (PR #25426)


@haram-KONA<https://github.com/haram-KONA> will be very happy if his hybrid was added CAR.KONA_HEV

File: selfdrive/car/hyundai/interface.py [hunk 1]
```diff
@@ -122,6 +122,8 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], disabl
       tire_stiffness_factor = 0.385
       ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0.], [0.]]
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
+      if candidate == CAR.KONA_EV_2022:
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/interface.py:125]:
```suggestion
      if candidate in (CAR.KONA_HEV, CAR.KONA_EV_2022):
```

**Comment by adeebshihadeh** [comment]
@haram-KONA can you get a route on this branch for us to look at?

**Comment by haram-KONA** [comment]
> @haram-KONA can you get a route on this branch for us to look at?
this route segment 

9a3aa85e86f9c3fc|2022-08-05--11-29-08--11

**Comment by haram-KONA** [comment]
kona hyberid 2020 - this route segment  : "  9a3aa85e86f9c3fc|2022-08-05--11-29-08--11  "
thank so much

**Comment by haram-KONA** [comment]
> @haram-KONA can you get a route on this branch for us to look at?

this is a "route segment" using the "kona-torque-tuning" branch.   (KOR 2022.08.14) 
I hope the data is appropriate.  thank you so much !!

route segment : 9a3aa85e86f9c3fc|2022-08-14--13-06-11--8

**Comment by haram-KONA** [comment]
Why isn't this branch merged into the master branch?
Maybe there's something wrong with that?
i am eagerly waiting for this branch to be merged into the Master......hehehe ...

**Pull request merged**